helm/minio/templates/_helper_create_svcacct.txt

```diff
@@ -91,7 +91,7 @@ connectToMinio $scheme
 {{- range $idx, $svc := .Values.svcaccts }}
 echo {{ tpl .accessKey $global }} > $MINIO_ACCESSKEY_SECRETKEY_TMP
 {{- if .existingSecret }}
-cat /config/secrets/{{ tpl .existingSecret $global }}/{{ tpl .existingSecretKey $global }} >> $MINIO_ACCESSKEY_SECRETKEY_TMP
+cat /config/secrets-svc/{{ tpl .existingSecret $global }}/{{ tpl .existingSecretKey $global }} >> $MINIO_ACCESSKEY_SECRETKEY_TMP
```

Comment:
this seems to be a breaking change here.

Reply:
This is related to the additional secret volumes in [this change](https://github.com/minio/minio/pull/16379/files#diff-258f84f473d006f234b01ecaffc90bdabe2addfd0eee08480356d850d8736c3eR75) - I use separate mount points for [user secrets](https://github.com/minio/minio/pull/16379/files#diff-258f84f473d006f234b01ecaffc90bdabe2addfd0eee08480356d850d8736c3eR66) and service secrets to avoid mount collisions when the same secret would be used for a user and a service account (okay, it shouldn't, but for test scenarios it could happen)

Reply:
Yeah but for existing users isn't it a problem?

Reply:
I'm not aware of any existing, previously working scenario: Without the change in `post-job.yaml` the secret's for `svcaccts` just havn't been mounted, so there was nothing. Using a secret mounted by the `users` object also wouldn't have worked as the path was different (missing `{{ tpl .existingSecret $global }}` in the path, see [_helper_create_user.txt](https://github.com/minio/minio/pull/16379/files#diff-f85addb2f442309143dcb4608f33e1db48c17fae8ea5c8659e083f5ffaeae353R96) what caused collitions when creating multiple users from different secrets having the same key name for the secret...

But when you are happy i could remove the `-svc` suffix for the `svcacct` secrets again, as this scenario causing a name / mount point collision between `users` and `svcacct` won't happen in my use case...